zkevm-circuits/src/evm_circuit/execution/return_revert.rs

```diff
@@ -64,6 +67,13 @@ impl<F: Field> ExecutionGadget<F> for ReturnRevertGadget<F> {
 
         cb.opcode_lookup(opcode.expr(), 1.expr());
 
+        // constrain op codes
+        cb.require_in_set(
```

Comment:
This check is not sufficient because it makes no difference between RETURN and REVERT. It will not be necessary when the check with `is_success` is re-enabled.

Reply:
@naure  what do you think any difference can be checking here except `is_success` ? I have no idea without it at the moment.

Reply:
It has to do with reversible writes that must be reverted. I can research more. If you prefer we can merge this PR and make another issue.

Reply:
OK, filed issue #721 to track it.